recipes/pygobject/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pygobject" %}
+{% set version = "3.25.1" %}
+{% set sha256 = "8728bb27078f7ee45c431ccbc237a58fb6d6eb1b7a0668bbe29eb107267bd736" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://ftp.gnome.org/pub/GNOME/sources/pygobject/3.25/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - gcc 4.8.*
```

Comment:
Sorry, I thought the idea was to depend on the defaults gcc rather than the container's one.